tests/testthat/test-code_feedback.R

```diff
@@ -14,6 +14,19 @@ test_that("code_feedback() returns a string if there are differences or NULL", {
   )
 })
 
+test_that("code_feedback() finds the closest match if multiple solutions", {
+  expect_equal(
+    code_feedback("a", gradethis_solutions("aa", "bb")),
+    "I expected `aa` where you wrote `a`."
+  )
+
+  # If there's a tie, the first option is selected
+  expect_equal(
+    code_feedback("a", gradethis_solutions("b", "c")),
+    "I expected `b` where you wrote `a`."
+  )
```

Comment:
See previous comments. Thinking about this again, we have three cases:

1. Tie for all solutions — use last
2. Tie among some solutions — use first? unless the last one is included?
3. Clear winner (by any amount) — use winner.

The middle case is the trickiest.

Reply:
[658ef2c](https://github.com/rstudio/gradethis/pull/289/commits/658ef2c58afd757863580ea0cc937e8c8f0d4371) implements this solution:
1. Tie for all solutions — use last
2. Tie among some solutions — use last
3. Clear winner (by any amount) — use winner